src/octoprint/plugins/events/templates/events_settings.jinja2

```diff
@@ -0,0 +1,65 @@
+<div class="alert alert-block">
+    <p>{{ _('For additional information on OctoPrint events please see the official documentation') }} <a href="https://docs.octoprint.org/en/master/events/index.html" target="_blank">{{ _('here') }}</a>.</p>
```

Comment:
This will not work in languages where "here" has to be in the middle of the translated sentence for grammatical reasons.

Instead:

```suggestion
    <p>{{ _('For additional information on OctoPrint events please see <a href="%(url)s" target="_blank" rel="noopener noreferer">the official documentation</a>.', url='https://docs.octoprint.org/en/master/events/index.html') }}</p>
```